recipes/etc-local.rb

```diff
@@ -30,6 +30,11 @@
       config_path = site['docroot']
     end
 
+    directory "#{config_path}/app/etc" do
+      recursive true
+      action :create
+    end
+
```

Comment:
btw, I just realised I don't like this any more, it'll change the permissions of the folder to root, when capistrano recipe previously changed it to deploy